lib/client/ood.js

```diff
@@ -154,7 +154,15 @@ function download() {
             if (isDir)
                 path	    = CloudCmd.PREFIX + '/oodzip' + path;
             else
-                path        = prefixUr + FS + path + '?download';
+                // Workaround for large files filling up /tmp folder
+                // uses a custom nginx route configured at /pun/download
+                if (ood_download) {
+                    path        = '/pun/download' + path + '?download=' + Date.now();
+                } else {
+                    path        = prefixUr + FS + path + '?download';
```

Comment:
Looks like it should be prefixed with `OOD` i.e. `OOD.ood_download`. In that file, whenever shell or file_editor are used they are prefixed that way.

https://github.com/OSC/cloudcmd/blob/61b7564a5dcc3d04defade9fb680984bfc672ca7/lib/client/ood.js#L57-L58

and

https://github.com/OSC/cloudcmd/blob/61b7564a5dcc3d04defade9fb680984bfc672ca7/lib/client/ood.js#L26